src/apps/lwaftr/podhashmap.lua

```diff
@@ -0,0 +1,304 @@
+module(..., package.seeall)
+
+local ffi = require("ffi")
+local C = ffi.C
+local band = require("bit").band
```

Comment:
I believe you meant to localize the `bit` module and after that localize `band`.